lib/yarn-auditer.js

```diff
@@ -51,16 +51,25 @@ function audit(config, reporter = reportAudit) {
       console.log(`Modules to whitelist: ${whitelist.join(', ')}.`);
     }
 
-    if (report) {
-      console.log('\x1b[36m%s\x1b[0m', 'Yarn audit report JSON-lines:');
+    if (report.full) {
+      console.log('\x1b[36m%s\x1b[0m', 'Yarn audit report JSON:');
+    }
+
+    if (report.summary) {
+      console.log('\x1b[36m%s\x1b[0m', 'Yarn audit report summary:');
     }
 
     function outListener(line) {
       const auditLine = JSON.parse(line);
       const { type, data } = auditLine;
-      if (report) {
+
+      if (report.full) {
         console.log(JSON.stringify(auditLine, null, 2));
       }
+      if (report.summary && auditLine.type === 'auditSummary') {
+        console.log(JSON.stringify(auditLine.data, null, 2));
```

Comment:
Looks like `type` and `data` are already in scope here.

Reply:
True, thanks, fixing this.

Reply:
Done